src/components/PublicPursuances/PublicPursuanceList.js

```diff
@@ -1,13 +1,48 @@
+// dumb component
+
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
 import * as postgrest from '../../api/postgrest';
 
 
 class PublicPursuanceList extends Component {
+
+  orderByDateDesc = (p1, p2) => {
+    p1["parsedDate"] = Date.parse(p1.created);
+    p2["parsedDate"] = Date.parse(p2.created);
+    return p2.parsedDate - p1.parsedDate;
```

Comment:
@thevaleriemack How about we call it `created_parsed`, so we know which date this is the parsed version of?